api/cotrip_app/views.py

```diff
@@ -1,3 +1,29 @@
-from django.shortcuts import render
+from rest_framework import generics, permissions
+from .serializers import CitySerializer, TopicSerializer
+from .models import City, Topic
+from django.views.generic import ListView, DetailView
+from django.views.generic.edit import CreateView, DeleteView, UpdateView
 
-# Create your views here.
+
+class CityList(generics.ListCreateAPIView):
+    queryset = City.objects.all()
+    serializer_class = CitySerializer
+    # permission_classes = (permissions.IsAuthenticated)
+
+
+class CityDetail(generics.RetrieveUpdateDestroyAPIView):
+    queryset = City.objects.all()
+    serializer_class = CitySerializer
+    # permission_classes = (permissions.IsAuthenticated)
```

Comment:
I don't think we want people to be able to delete or update these through the API. Same with Create. We probably want to limit the functionality to list and create.